src/frameworks/internal/messenger/messenger.js

```diff
@@ -179,6 +183,7 @@ cwc.framework.Messenger.prototype.postMessage = function(name, value) {
   this.appWindow.postMessage({
     'name': name,
     'value': value,
+    'token': this.token_,
```

Comment:
I'll remove it.

It allows multiple cwc.framework.Messenger objects to listen on the same window object and know which messages to ignore. It solved a problem I was having with the sphero, messanger and preview unit tests all instantiating messenger objects. However, I plan to move the preview tests to core per your suggestion, so that likely won't be an issue.